Header header

# GPIO values for the 3.3V processor.  -1 indicates that
# It has not been read, 0 if it is at a logic low level and
# 1 for a logic high level
int8 gpio33_0
int8 gpio33_1
int8 gpio33_2
int8 gpio33_3
int8 gpio33_4
int8 gpio33_5

# GPIO values for the 5.0V processor.  -1 indicates that
# It has not been read, 0 if it is at a logic low level and
# 1 for a logic high level
int8 gpio5_0
int8 gpio5_1
int8 gpio5_2
int8 gpio5_3
int8 gpio5_4
int8 gpio5_5